services/movie.py

```diff
@@ -1,11 +1,13 @@
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: list[int] = None,
```

Comment:
if default is None
it already optional parameter 
change typing in other places the same

Reply:
its done, please check

Reply:
Not done 
<img width="393" alt="image" src="https://user-images.githubusercontent.com/65166203/220574311-30027ae5-3d45-407f-8535-bb639646d26a.png">
The same in other places